### PR TITLE
Changing overlay layout to a single column

### PR DESCRIPTION
**Pull request by rhamilto**
localhost-9000-dev-console-
localhost-9000-dev-console- iphone 6

### CODE REVIEW

File: src/components/create-from-builder/create-from-builder-info.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<div class="order-service-details">
+  <div class="order-service-details-top">
+    <div class="service-icon">
+      <span class="icon {{$ctrl.imageStream.iconClass}}"></span>
```
Comment (spadgett) [inline src/components/create-from-builder/create-from-builder-info.html]: aria-hidden="true"

File: src/components/create-from-builder/create-from-builder.controller.ts [hunk 2]
```diff
@@ -184,6 +195,12 @@ export class CreateFromBuilderController implements angular.IController {
     }
   };
 
+  private showInfo = () => {
+    this.clearValidityWatcher();
+    this.ctrl.nextTitle = 'Next >';
+    this.reviewStep.allowed = true;
```
Comment (spadgett) [inline src/components/create-from-builder/create-from-builder.controller.ts]: This line looks wrong. I think we can just delete it.

File: src/components/create-from-builder/create-from-builder.html [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <div class="order-service">
   <pf-wizard
-       hide-header="true"
+       title="{{$ctrl.imageStream.name}} {{$ctrl.istag.name}}"
```
Comment (spadgett) [inline src/components/create-from-builder/create-from-builder.html]: Can you check that this updates correctly when you pick different versions in the dropdown inside the wizard?

File: src/components/order-service/order-service-info.html [hunk 4]
```diff
@@ -0,0 +1,32 @@
+<div class="order-service-details">
+  <div class="order-service-details-top">
+    <div class="service-icon">
+      <span ng-if="!$ctrl.imageUrl" class="icon {{$ctrl.iconClass}}"></span>
```
Comment (spadgett) [inline src/components/order-service/order-service-info.html]: aria-hidden="true"

File: src/components/order-service/order-service-info.html [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+        {{$ctrl.serviceName}}
+      </div>
+      <div ng-if="$ctrl.serviceClass.tags" class="order-service-tags">
+        <span ng-repeat="tag in $ctrl.serviceClass.tags" class="tag">
+          {{tag}}
+        </span>
+      </div>
+      <div ng-if="$ctrl.serviceClass.resource.externalMetadata.documentationUrl" class="order-service-documentation-url">
+        <a ng-href="{{$ctrl.serviceClass.resource.externalMetadata.documentationUrl}}" target="_blank" class="learn-more-link">Learn More <i class="fa fa-external-link" aria-hidden="true"></i></a>
+      </div>
```
Comment (spadgett) [inline src/components/order-service/order-service-info.html]: We should probably add `externalMetadata.supportUrl` too

File: src/components/create-from-builder/create-from-builder.html (hunk 3 above)
Comment (rhamilto) [reply src/components/create-from-builder/create-from-builder.html]: Confirmed that it does.  :)

File: src/components/order-service/order-service-info.html (hunk 5 above)
Comment (rhamilto) [reply src/components/order-service/order-service-info.html]: screen shot 2017-09-21 at 12 08 47 pm

**Comment by rhamilto** [comment]
@spadgett's comments addressed.  Thanks!

**Comment by rhamilto** [comment]
@jeff-phillips-18, can you take a look today?  @spadgett is hoping we can this merged so we can also merge https://github.com/openshift/origin-web-console/pull/2127  today.

**Comment by jeff-phillips-18** [comment]
@rhamilto yeah, catching up with reviews now. Sorry for the delay.

**Pull request merged**